appendix-basic65.tex

```diff
@@ -4138,7 +4141,7 @@ \subsection{FORMAT}
 
                   {\bf I} disk ID. The maximum length is 2 characters.
 
-                  {\bf diskname} is either a quoted string, e.g. \screentext{"DATA"} or a string expression in brackets, e.g. \screentext{(DN\$)}. The maximum length of is 16 characters.
+                  {\bf diskname} is either a quoted string, e.g. \screentext{"DATA"} or a string expression in brackets, e.g. \screentext{(DN\$)} The maximum length of is 16 characters.
```

Comment:
Would need revision to be consistent with the above.

If it's too difficult to write around it, we can just leave the period in place in cases like this.